.yamllint

```diff
@@ -4,16 +4,26 @@ extends: default
 rules:
   braces:
     level: warning
+    min-spaces-inside: 1
+    max-spaces-inside: 64
   brackets:
     level: warning
   colons:
     level: warning
   commas:
     level: warning
+    min-spaces-after: 1
+    max-spaces-after: 64
   indentation:
     level: warning
   line-length:
-    max: 200
+    max: 250
   truthy:
     level: error
-
+    allowed-values:
+      - 'false'
+      - 'true'
+      - 'no'
+      - 'yes'
+      - 'on'
```

Comment:
"on" is a key in gha workflows (see near top of file).
Since we're not running yamllint here, I am going to abandon those changes and focus on coverity.